doc/source/multivisit.rst

```diff
@@ -3,4 +3,435 @@
+PanSTARRS project as described at `PanSTARRS Sky tessellation patterns
+<https://outerspace.stsci.edu/display/PANSTARRS/PS1+Sky+tessellation+patterns>`_.
+
+.. figure:: images/figure_aitoff.png
+  :figwidth: 90%
+  :alt: Aitoff plot of all 2,009 PS1 projection cells for the 3PI survey.
+
+  Aitoff plot of all 2,009 PS1 projection cells for the 3PI survey.  The coverage extends from declination −30° to the
+  north celestial pole.
+
+
+.. figure:: images/figure_pole.png
+  :figwidth: 90%
+  :alt: PS1 projection cells near the north celestial pole.
+
+  PS1 projection cells near the north celestial pole, where the image overlap is greatest due to convergence of the RA grid.
+  The projection cells are 4°x4° in size and are on rings spaced by 4° in declination.
+
+MVM processing uses these pre-defined projection cells after extending them to cover the entire sky
+with cells that are actually 4.2°x4.2° in size and are on rings spaced by 4° apart in declination.  This provides
+sufficient overlap to allow data from one projection cell to be merged with data from a neighboring cell if necessary.
+
+All these definitions, including finely each ProjectionCell should be divided to define the SkyCells, are encoded in a
+FITS table installed as part of the `drizzlepac` package:
+
+    drizzlepac/pars/allsky_cells.fits
+
+
+Defining each SkyCell
+----------------------
+Each projection cell is split into a grid of 21 x 21 'sky cells' which serves as the most basic MVM product generated
+during MVM processing.  Sky cells are approximately 0.2degrees x 0.2degrees in size (~21500 x ~21500 pixels) and
+they have the same WCS as the ‘projection cell’.  Each skycell gets identified by its position within the projection cell
+as shown in this figure:
+
+.. figure:: images/SkyCell_numbering.png
+  :width: 90%
+  :alt: Numbering convention for SkyCells within a Projection Cell.
+
+  Numbering convention for SkyCells within a Projection Cell used for naming the SkyCell.
+
+This provides a way to uniquely identify any position on the sky that can be used as the basis for a unique filename for
+all products generated from all the expsosures that overlap each SkyCell.
+
+The WCS for each SkyCell gets defined as a subarray of the Projection cell's WCS.  This allows data across SkyCells in
+the same projection cell to be combined into larger mosaics as part of the same tangent plane without performing any
+additional resampling.
+
+Code for Defining SkyCell ID
+'''''''''''''''''''''''''''''
+The code for interfacing with the cell definitions table can be imported in Python using:
+
+.. code:: python
+
+   from drizzlepac.haputils import cell_utils
+
+Determining what sky cells overlap any given set of exposures on the sky can be done using the function:
+
+.. code:: python
+
+  sky_cells_dict = cell_utils.get_sky_cells(visit_input, input_path=None)
+
+where **visit_input** is the Python list of filenames of exposures.
```

Comment:
Clarify if this a list of pipeline FLT/Cs or SVM FLT/Cs or ???

Reply:
Changed in 241a672